### PR TITLE
feat: `keydown` event

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* added `keydown` event
* modified `editingfinish` event
  * added editing info property to know that whether to save the value or be triggered by key

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다.

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Review by jajugoguma — APPROVED** [review]
리뷰 완료합니다!

File: packages/toast-ui.grid/src/view/editingLayer.tsx [hunk 1]
```diff
@@ -65,19 +65,39 @@ export class EditingLayerComp extends Component<Props> {
...
-        dispatch('setValue', rowKey, columnName, value);
-      }
-      if (isFunction(this.editor.beforeDestroy)) {
+      const { rowKey, columnName, value } = this.getEditingCellInfo();
+
+      dispatch('setValue', rowKey, columnName, value);
+
+      if (this.editor?.beforeDestroy) {
         this.editor.beforeDestroy();
       }
```
Comment (lja1018) [inline packages/toast-ui.grid/src/view/editingLayer.tsx:98]: `this.editor?.beforeDestroy?.();`는 어떠신가요?

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by adhrinae — COMMENTED** [review]
리뷰 완료합니다.

**Pull request merged**